sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -20,6 +20,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" />
     <PackageReference Include="System.Buffers" />
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" />
```

Comment:
BTW, we need to use DiagSource 4.6.0 preview, it uses W3C trace-context  and we should never use old Id format in Azure SDKs

Reply:
Isn't the format supposed to be controlled by the app?

Reply:
Ah, I see, we need to know about TraceStateString too. https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/src/System/Net/Http/DiagnosticsHandler.cs#L198